website/js/sketch.js

```diff
@@ -1,6 +1,5 @@
 function preload () {
   loadedTags = loadJSON("https://<API-GATEWAY-ADDRESS>/<STAGE>/getTagsForWishTree");
-  //console.log(loadedTags);

```

Comment:
The removal of the comments / debug messages in this one are good and make sense!